src/pages/[platform]/build-a-backend/data/custom-business-logic/search-and-aggregate-queries/index.mdx

```diff
@@ -148,13 +148,24 @@ const openSearchDomain = new opensearch.Domain(
   {
     version: opensearch.EngineVersion.OPENSEARCH_2_11,
     nodeToNodeEncryption: true,
+    // set removalPolicy to DESTROY for sandbox environemnt
+    // removalPolicy: RemovalPolicy.DESTROY,
```

Comment:
Can we uncomment this line? If someone copy-paste the code, they may forget to change the removal policy.
```suggestion
    // set removalPolicy to RETAIN for production environment
    removalPolicy: RemovalPolicy.DESTROY,
```